LibAddonMenu-2.0/controls/panel.lua

```diff
@@ -48,7 +48,10 @@ local function ForceDefaults(panel)
 
     for i = 1, #panelControls do
         local updateControl = panelControls[i]
-        if updateControl.UpdateValue and updateControl.data.default ~= nil then
+        local updateControlData = updateControl.data
+        if updateControlData.resetFunc and type(updateControlData.resetFunc) == "function" then
+            updateControlData.resetFunc()
```

Comment:
I think it would also make sense to pass updateControl to resetFunc, otherwise it won't be easy to access the control.